gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/saf/SafProviderBeansConfig.java

```diff
@@ -1,32 +0,0 @@
-/*
- * This program and the accompanying materials are made available under the terms of the
- * Eclipse Public License v2.0 which accompanies this distribution, and is available at
- * https://www.eclipse.org/legal/epl-v20.html
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Copyright Contributors to the Zowe Project.
- */
-package org.zowe.apiml.gateway.security.service.saf;
-
-import lombok.RequiredArgsConstructor;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.client.RestTemplate;
-import org.zowe.apiml.gateway.security.service.AuthenticationService;
-import org.zowe.apiml.passticket.PassTicketService;
-
-@Configuration
-@RequiredArgsConstructor
-public class SafProviderBeansConfig {
```

Comment:
Why do you believe that conditional creation annotated on bean is improvement over config bean that has place for multiple values on one place?

Reply:
There is no right answer, I guess. Maybe, unless you need a code to invoke except the calling of constructor the configuration bean is overhead.

Reply:
I prefer only wrapping vanilla Java code with a bean. For native Spring classes, it seems better to use Component/Service. It makes the code more readable and understandable.

Reply:
Then please keep it this way as this is established pattern that we also use within Caching Service and there is no strong need to change it. 

Or we can have the discussion and then change the Caching Service as well.